src/main/java/org/kpmp/spatialViewerDataset/SpatialViewerDatasetService.java

```diff
@@ -48,6 +50,10 @@ public void setErrors(List errors) {
 		}
 	}
 
+    public SpatialViewerDatasetService() {
```

Comment:
Spring should have nothing to complain about. You should be able to have an overridden constructor with parameters without defining a default constructor.